shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/TxData.hs

```diff
@@ -268,15 +299,23 @@ data TxIn crypto
 
 instance NoUnexpectedThunks (TxIn crypto)
 
--- | The output of a UTxO.
+-- |The output of a Tx.
 data TxOut crypto
-  = TxOut !(Addr crypto) !Coin
-  deriving (Show, Eq, Generic, Ord)
+  = TxOut !(Addr crypto) !(Value crypto)
```

Comment:
If we follow my second suggestion for the `CompactValue` property then this should just be `CompactValue` and you can ditch `UTxOOut`.

Reply:
Value and CompactValue need to be different things - but I believe that this might have to change because of Shelley changes, which will introduce a consistent way of using "compact" things in the UTxO